client/core/bookie.go

```diff
@@ -191,6 +191,13 @@ func (b *bookie) logEpochReport(note *msgjson.EpochReportNote) error {
 		return fmt.Errorf("epoch report has zero-valued candle end stamp")
 	}
 
+	marketID := marketName(b.base, b.quote)
+	b.SetRecentMatches(note.MatchesSummary, note.EndStamp)
+	b.send(&BookUpdate{
+		Action:   "epoch_report",
```

Comment:
This action would need to be defined in a const, near where `FreshBookAction` is defined.  But let's not use `"epoch_report"` since the recent matches is just a subset of the epoch report message from the server.  Perhaps `epoch_match_summary`